CHANGELOG.md

```diff
@@ -1,8 +1,309 @@
 ## Unreleased
 
+TODO: Structopt and traits
+TODO: `Shell`, `YamlLoader`
+TODO: `cargo`, `std` features
+
+#### BREAKING CHANGES
+
+* **Removed**
+  * `From<&yaml_rust::yaml::Hash>` for `ArgGroup`
+  * **Macros**
+    * `_clap_count_exprs`
+* **Renamed Methods**
+  * `App::set_term_width` => `App::term_width`
+  * `Arg::from_yaml` => `Arg::from`
+  * `Arg::with_name` => `Arg::new`
+  * `ArgGroup::from_yaml` => `ArgGroup::from`
+  * `ArgGroup::with_name` => `ArgGroup::new`
+
+#### Features
+
+* Added `Indices` that is returned by `ArgMatches::indices_of`
+* **Added Methods**
+  * **Arg**
+    * `Arg::default_missing_value`
+    * `Arg::default_missing_value_os`
+    * `Arg::default_missing_values`
+    * `Arg::default_missing_values_os`
+    * `Arg::short_alias`
+    * `Arg::short_aliases`
+    * `Arg::visible_short_alias`
+    * `Arg::visible_short_aliases`
+* **Added Variants**
+  * `AppSettings::DisableHelpFlags`
+
+#### Enhancements
+
+* `help_heading` defined on `Arg` now has higher priority than `App`
+* Limited default text wrapping to 100 when `wrap_help` feature is not enabled
+* Multiple bug fixes and error message improvements
+
+
+<a name="v3.0.0-beta.1"></a>
```

Comment:
Why do we have these `<a name=...>` tags? They don't seem to do anything at all.

Reply:
Dunno. I just copied it from the previous sections. We can look into it later